ena_upload/templates/ENA_template_ERC000011_sample.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?python
+import pandas as pd
+def attributetest(row, column):
+    if hasattr(row, column) and pd.notna(row[column]) and not row[column].isspace() and str(row[column]).lower() not in ['nan', 'na']:
+        return True 
+    else:
+        return False?>
+<SAMPLE_SET xmlns:py="http://genshi.edgewall.org/"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="ftp://ftp.sra.ebi.ac.uk/meta/xsd/sra_1_6/SRA.sample.xsd">
+    <py:for each="index, row in df.iterrows()">
+    <SAMPLE alias="${row.alias}" center_name="${center}">
+        <!--TODO: UNIQUE NAME FOR SAMPLE-->
+        <!--TODO: CENTER NAME-->
+        <TITLE>${row.title}</TITLE>
+        <!--TODO: A SHORT INFORMATIVE DESCRIPTION OF THE SAMPLE-->
+        <SAMPLE_NAME>
+            <TAXON_ID>${row.taxon_id}</TAXON_ID>
+            <!--TODO: PROVIDE NCBI TAXID FOR ORGANISM (e.g. 9606 for human)-->
```

Comment:
@rabuono are you planning to address those TODOs before we merge?

Reply:
Those TODOs were already in the original [ENA_template_vir_sample.xml](https://github.com/usegalaxy-eu/ena-upload-cli/blob/159475723974f4b85f22574866701055ff0a7984/ena_upload/templates/ENA_template_vir_sample.xml) from which I based this new one on. I don't know what was the original plan on how to address them.
So, in short, no.